### PR TITLE
Fix/travis

### PR DESCRIPTION
**Pull request by papandreou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "serve": "*",
     "sinon": "1.17.7",
     "unexpected": "10.23.0",
-    "unexpected-documentation-site-generator": "^4.2.3",
-    "unexpected-markdown": "1.4.1"
+    "unexpected-documentation-site-generator": "4.5.0",
```
Comment (joelmukuthu) [inline package.json]: I think @sunesimonsen wanted the lax version range so plugins get the latest style updates whenever they are added to the site generator :)

Comment (sunesimonsen) [inline package.json]: Correct.

Comment (sunesimonsen) [inline package.json]: I trust myself when it comes to backwards compatibility.

Comment (sunesimonsen) [inline package.json]: But no worries, it is fine. I did it while I was doing a lot of updates to the documentation generators.

Comment (papandreou) [inline package.json]: Oh, sorry, that was an accident. Fixed in https://github.com/unexpectedjs/unexpected-sinon/commit/745bc2d74fe776fab792e32a270efa37d5c1b567

**Pull request merged**